src/main/java/domain/Attendances.java

```diff
@@ -0,0 +1,40 @@
+package domain;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Attendances {
+    private final List<Attendance> attendances;
+
```

Comment:
```suggestion
public class Attendances {

    private final List<Attendance> attendances;

```
다른클래스의 컨밴션과 맞추어 한 줄 띄워줍시다!

Reply:
엇 컨벤션을 통일시키려고 신경썼는데 확인을 못한 부분이 있었네요 😅 주의하겠습니다!